plugins/MauticCloudStorageBundle/Form/Type/OpenStackType.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- * @copyright   2014 Mautic Contributors. All rights reserved
+ * @copyright   2019 Mautic Contributors. All rights reserved
```

Comment:
Same here.

Oh, basically everywhere.

Reply:
ok, I think I saw that in another PR, got it, no more changes in the years.

Reply:
Yea, new files get the current year. But existing files keep their original copyright year.